src/utils/system.rs

```diff
@@ -121,7 +121,11 @@ pub fn print_error(err: &Error) {
     if let Some(clap_err) = err.downcast_ref::<clap::Error>() {
         clap_err.exit();
     }
-    eprintln!("{:?}", err);
+
+    eprintln!("{} {}", style("error:").red(), err);
+    err.chain()
+        .skip(1)
+        .for_each(|cause| eprintln!("  {} {}", style("caused by:").dim(), cause));
```

Comment:
Should we extract this into a helper? And use whenever we want to log an error out?

Reply:
It's only intended for `anyhow` errors, which will always bubble up to this place